config/config.go

```diff
@@ -96,6 +96,11 @@ func (cfg *Config) SetRoot(root string) *Config {
 	return cfg
 }
 
+func (cfg *Config) SetPrivValidatorListenAddr(addr string) *Config {
```

Comment:
I'd prefer not to have this function. It does not bring any value and encourages mutation.

Reply:
Any pointers as to how I might override the priv_val_laddr config when testing option without it?